docs/runbooks/VMStorageClassWarning.md

```diff
@@ -23,18 +23,18 @@ running the following PromQL query:
 <!--DS: You can use the Openshift metrics explorer available at 'https://{OPENSHIFT_BASE_URL}/monitoring/query-browser'.-->
 
 ```promql
-$ kubevirt_ssp_vm_rbd_volume{rxbounce_enabled="false", volume_mode="Block"} == 1
+$ kubevirt_ssp_vm_rbd_block_volume_without_rxbounce == 1
```

Comment:
@machadovilaca  Please update this so that this line that should be copied doesn't include also the line below

Reply:
@machadovilaca  I think we should either state this should be run in the OCP console->observe->metrics page or as a curl command

Reply:
fixed the first comment

we already have a message to use the metrics explorer above: `You can use the Openshift metrics explorer available at 'https://{OPENSHIFT_BASE_URL}/monitoring/query-browser'`

Reply:
oh, its a d/s comment only... What about upstream? 
It should say they should query Prometheus for this metric

Reply:
updated